medical_disease/models/medical_pathology.py

```diff
@@ -1,9 +1,7 @@
 # -*- coding: utf-8 -*-
 # #############################################################################
 #
-# Tech-Receptives Solutions Pvt. Ltd.
-# Copyright (C) 2004-TODAY Tech-Receptives(<http://www.techreceptives.com>)
-# Special Credit and Thanks to Thymbra Latinoamericana S.A.
+# Copyright 2008 Luis Falcon <lfalcon@gnusolidario.org>
```

Comment:
Question: did you validate with him the date and copyright information?

Reply:
They will not speak to us like human beings unfortunately. This is the date of inception for GNUHealth, so it's the date I put.

Reply:
(It's the info here too- https://sourceforge.net/projects/medical/files/Oldfiles/1.0.1)